test/binary/bad-elem-flags-no-reftype.txt

```diff
@@ -0,0 +1,19 @@
+;;; TOOL: run-gen-wasm-bad
```

Comment:
I probably should start adding explicit `--disable-reference-types` so we don't have to change it after the extension gets enabled by default.